### ISSUE TITLE
Changing language of the spellchecker and turning it on/off.

### ISSUE THREAD

**Issue by nilsfr** [lbryio/lbry-desktop#6108]
I start the Lbry desktop app for Linux with a Norwegian Bokmål (nb_NO) locale. When I comment a spellchecker with this locale is used. I, however, frequently write comments in English which leads to many red squiggly lines.

I would like to be able to change the language of the spellchecker and also be able to turn it on and off.

Perhaps it can be done using what is [described in the Electron documentation](https://www.electronjs.org/docs/tutorial/spellchecker)?

**Comment by kauffj** [comment]
Do you experience this in your operating system? Is your operating system set to both languages? I suspect that could be read into the Electron setting, if so

**Comment by nilsfr** [comment]
Yes, I do have multiple locale for which nb_NO is the default. I would like to be able to use the context menu for the comment textarea to be able to change the language of the spellchecker and to be able to turn it on and off. I think this is useful for many non-native English speakers.

I use Linux and `locale -a` gives:
```
C
C.UTF-8
en_AG
en_AG.utf8
en_AU.utf8
en_BW.utf8
en_CA.utf8
en_DK.utf8
en_GB.utf8
en_HK.utf8
en_IE.utf8
en_IL
en_IL.utf8
en_IN
en_IN.utf8
en_NG
en_NG.utf8
en_NZ.utf8
en_PH.utf8
en_SG.utf8
en_US.utf8
en_ZA.utf8
en_ZM
en_ZM.utf8
en_ZW.utf8
nb_NO.utf8
POSIX
```

**Comment by kauffj** [comment]
Thanks @nilsfr. We'll look at this but want to find a way to make it work by default, since once it requires fiddling with settings we've lost 80% of users (that doesn't mean it can't be a setting as well).
